configure.ac

```diff
@@ -1893,25 +1859,14 @@ AC_MSG_NOTICE([Linux Netfilter support requested: ${enable_linux_netfilter:=auto
 dnl Look for libnetfilter_conntrack options (needed for QOS netfilter marking)
 SQUID_AUTO_LIB(netfilter-conntrack,[Netfilter conntrack],[LIBNETFILTER_CONNTRACK])
 AC_MSG_NOTICE([Linux Netfilter Conntrack support requested: ${with_netfilter_conntrack:=auto}])
```

Comment:
FWIW, this line seems inconsistent with other SQUID_CHECK_LIB_WORKS() calls. I do not insist on (or even suggest) this AC_MSG_NOTICE() removal, but please check whether it should be kept despite this inconsistency. If it should be kept, perhaps we should add a source code comment to document why, but I do not insist on adding that comment either.

Reply:
Removed.